svgdigitizer/electrochemistry/electrolyte.py

```diff
@@ -0,0 +1,197 @@
+    'NBIIXXVUZAFLBC-UHFFFAOYSA-N': {'name': 'H3PO4', 'components': [Acid(pKa=[2.16, 7.21, 12.32], charge=0)]},  # CRC handbook of chemistry and physics 2017
+    'BDAGIHXWWSANSR-UHFFFAOYSA-N': {'name': 'HCOOH', 'components': [Acid(pKa=[3.75], charge=0)]},  # CRC handbook of chemistry and physics 2017
+    'QTBSBXVTEAMEQO-UHFFFAOYSA-N': {'name': 'acetic acid', 'components': [Acid(pKa=[4.78], charge=0)]},
+    'VEXZGXHMUGYJMC-UHFFFAOYSA-N': {'name': 'HCl', 'components': [Acid(pKa=[-6.2], charge=0)]},  # Analytical Chemistry, 43(7), 1971, S. 969–970.
+    'JUJWROOIHBZHMG-UHFFFAOYSA-N': {'name': 'pyridine', 'components': [Acid(pKa=[5.23], charge=1)]},  # CRC handbook of chemistry and physics 2017
+    'QGZKDVFQNNGYKY-UHFFFAOYSA-N': {'name': 'NH3', 'components': [Acid(pKa=[9.25], charge=1)]},  # CRC handbook of chemistry and physics 2017
+    'BVKZGUZCCUSVTD-UHFFFAOYSA-N': {'name': 'H2CO3', 'components': [Acid(pKa=[6.35, 10.33], charge=0)]},  # CRC handbook of chemistry and physics 2017
+    'HEMHJVSKTPXQMS-UHFFFAOYSA-M': {'name': 'NaOH', 'components': [Neutral(charge=1)]},
+    'FAPWRFPIFSIZLT-UHFFFAOYSA-M': {'name': 'NaCl', 'components': [Acid(pKa=[-6.2], charge=0), Neutral(charge=1)]},
+    'PMZURENOXWZQFD-UHFFFAOYSA-L': {'name': 'Na2SO4', 'components': [Acid(pKa=[-3, 1.9], charge=0), Neutral(charge=2)]},
+    'RYFMWSXOAZQYPI-UHFFFAOYSA-K': {'name': 'Na3PO4', 'components': [Acid(pKa=[2.16, 7.21, 12.32], charge=0), Neutral(charge=3)]},
+    'BNIILDVGGAEEIG-UHFFFAOYSA-L': {'name': 'Na2HPO4', 'components': [Acid(pKa=[2.16, 7.21, 12.32], charge=0), Neutral(charge=2)]},
+    'AJPJDKMHJJGVTQ-UHFFFAOYSA-M': {'name': 'NaH2PO4', 'components': [Acid(pKa=[2.16, 7.21, 12.32], charge=0), Neutral(charge=1)]},
+    }
+
+
+class Electrolyte():
+    r"""
+    Electrolyte
+    """
+    def __init__(self, electrolyte):
+        self.electrolyte = electrolyte
+
+    @property
+    @cache
+    def pH(self):
+        r"""
+        pH of the electrolyte.
+        """
+        try:
+            return self.electrolyte['pH']
+        except KeyError:
+            return self.estimate_pH()
+
+    def estimate_pH(self):
+        r"""
+        Estimates the pH of aquoues electrolytes.
+
+        EXAMPLES::
+
+            CID can be retrieved with InChIKey::
+            >>> from svgdigitizer.electrochemistry.electrolyte import Electrolyte
+            >>> electrolyte_dict = {"type": "aq", "components": [{"name": "H2SO4","type": "acid", "concentration": {"value": 0.1, "unit": "M"}}, {"name": "Na2SO4","type": "salt", "concentration": {"value": 0.1, "unit": "M"}}]}
+            >>> electrolyte = Electrolyte(electrolyte_dict)
+            >>> electrolyte.pH
+            1.3538681030273414
+        """
+
+        if self.electrolyte['type'] != 'aq':
+            raise NotImplementedError("pH calculations are only implemented for aqueous solutions!")
```

Comment:
Maybe this should be checked before invoking `Electrolyte` in the `__main__.cv`.  
Then you don't need to raise an error, but only add pH if the system is aqueous.

Reply:
I am not sure, as ```Electrolyte``` could also be invoked outside ```__main__.py```.

Reply:
I guess then we have to double-check since if we want to digitize non-aqueous CVs the program will always stop here.

From the dict above, I assume it would be sufficient to basically call the class `Components`, which has a property called pH, which is calculated from the components and then added to the metadata file.

Reply:
The estimate can also be `None` in this case.

Reply:
i.e., don't raise an exception but just return `None`.

Reply:
I agree.